deploy-manage/deploy/cloud-enterprise/configure.md

```diff
@@ -16,3 +19,17 @@ Now that you have Elastic Cloud Enterprise up and running, take a look at some o
 * [Change allocator disconnect timeout](change-allocator-disconnect-timeout.md) - Configure how long ECE waits before considering allocators to be disconnected.
 * [Migrate ECE on Podman hosts to SELinux in enforcing mode](migrate-ece-on-podman-hosts-to-selinux-enforce.md) - Migrate ECE to SELinux in `enforcing` mode using Podman.
 
+## Administering your installation [ece-administering-ece]
+
+Now that you have Elastic Cloud Enterprise up and running, take a look at the things you can do to keep your installation humming along, from adding more capacity to dealing with hosts that require maintenance or have failed. They are all presented in the [](../../maintenance.md) section.
```

Comment:
I can tell that you didn't 100% work on this page but this para is duplicative to the one at the top of the page, and points to the maintence section, where these page are not. ignore if this is being worked on later :)

Reply:
exactly, I want to leave the duplicate paragraph for the moment as it shows better from where everything came.

This will be in my next PR. I'll add a work in progress header to the page :)